packages/validation/README.md

```
@@ -0,0 +1,110 @@
+});
+
+// Offset pagination
+const offsetInput = offsetPaginationSchema.parse({
+  page: 1,
+  limit: 20,
+});
+```
+
+## Validation Functions
+
+### validate()
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)